Formula/kumitateru.rb

```diff
@@ -0,0 +1,21 @@
+# typed: false
+
+# frozen_string_literal: true
+
+# Kuitateru formula
+class Kumitateru < Formula
+  desc "Build system for Garmin ConnectIQ. Simple, fast, powerful"
+  homepage "https://github.com/ggoraa/kumitateru"
+  url "https://github.com/ggoraa/kumitateru/releases/download/0.4.0/kumitateru-mac"
```

Comment:
This should be built from source. See: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae